src/module/actor/actor-sheet.js

```diff
@@ -187,7 +187,7 @@ export class OseActorSheet extends ActorSheet {
     let element = event.currentTarget;
     let attack = element.parentElement.parentElement.dataset.attack;
     const rollData = {
-      actor: isNewerVersion(game.version, "10.264") ? this : this.data, //v9-compatibility
+      actor: this,
```

Comment:
`this` is `OseActorSheetCharacter`. `_rollAttack()` is only used for "improvised melee" and "improvised ranged" (by clicking on the MEL and MIS headers of the melee and missile bonus boxes).

Because it appeared `targetAttack()` will take either one for the roll data without complaint, I looked into the reason...it's completely ignored

removed `rollData` as irrelevant in my third commit today